README.md

```diff
@@ -15,65 +15,113 @@ Base class for [orbit-db](https://github.com/orbitdb/orbit-db) data stores. You
 
 - Node.js >= 8.0.0
 
-### API
```

Comment:
I would keep the API heading, as a second-level heading.

Reply:
nice catch 👍 updated.

Reply:
Could you... add a TOC? :D

Reply:
Could I... borrow your TOC generator?